openshift/template_insights-inventory.yaml

```diff
@@ -182,10 +189,10 @@ parameters:
   - description: 'The inventory service path prefix. Used to build the API url (/<INVENTORY_PATH_PREFIX>/<INVENTORY_APP_NAME>/api/v1/)'
     displayName: Inventory Path Prefix
     name: INVENTORY_PATH_PREFIX
-    value: /r/insights/platform
+    value: r/insights/platform
     required: false
   - description: 'The inventory service app name. Used to build the API url (/<INVENTORY_PATH_PREFIX>/<INVENTORY_APP_NAME>/api/v1/)'
```

Comment:
This URL fragment needs updating here and for `INVENTORY_PATH_PREFIX` as otherwise, if you interpret the description, the path becomes `r/insights/platform/inventory/api/api/v1`.

Reply:
ugh there's more wrong with this thought... I'll get this fixed up.

As a side note, something I want to bring up to Derek is that if you specify a lead / then a deploy breaks.